### PR TITLE
Implement I18n for internationalization support

### PR DESCRIPTION
**Pull request by CristianOliveiraDaRosa**
We needed in our project(TorrePlay) create some notification descriptions in another language because we will use the default description for our app notifications. Then for do that we implemented I18n for internationalization support.

Changes:
We created the en.yml file on "config/locales" and changed the strings on judge.rb.

We liked a lot this gem it is being useful for us!
Thanks!

### CODE REVIEW

**Comment by tute** [comment]
This is great, thanks!

Can you please make them named interpolations? http://guides.rubyonrails.org/i18n.html#interpolation

Thanks again!

**Comment by CristianOliveiraDaRosa** [comment]
Thanks for the rapid answer.
I will do as your sugested.

Please let me know if I should change others strings in others files.

File: config/locales/en.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+en:
+  merit:
+    granted_points: "granted %{points} points"
+    granted_badge: "granted %{badge_name} badge"
+    removed_badge: "removed %{badge_name} badge"
```
Comment (tute) [inline config/locales/en.yml]: Nitpick: let's sort them alphabetically.

File: README.md [hunk 2]
```diff
@@ -292,6 +292,18 @@ end
...
+### I18n
+
+Merit uses default messages with I18n for notify alerts. To customize your app, you can set up your locale file:
+
+```yaml
+en:
+  merit:
+    granted_points: "granted %{points} points"
+    granted_badge: "granted %{badge_name} badge"
+    ...
```
Comment (tute) [inline README.md]: Given we only have three translations for the entire project, we can list them all here.

File: README.md [hunk 3]
```diff
@@ -292,6 +292,18 @@ end
 config.add_observer 'ReputationChangeObserver'
 ```
 
+### I18n
```
Comment (tute) [inline README.md]: This should be a top-level heading, only one pound sign.

**Comment by tute** [comment]
Looking great, thank you! Left a few small comments, please afterwards squash the commits together, and ready to merge!

**Comment by tute** [comment]
HEy @CristianOliveiraDaRosa! What happened with your commits?

**Comment by CristianOliveiraDaRosa** [comment]
Sorry @tute I tried to do a rebase to squash the commits but for some reason it didn't work. So I needed  to do by the hard way. I created this new PR #218

**Pull request closed without merging**